schema/deploy/shipping/views.sql

```diff
@@ -3283,10 +3286,43 @@ create or replace view shipping.__uw_priority_queue_v1 as (
         -- CHR 97 is 'a'
         -- DOW is 0 through 6, starting on Sunday; therefore, Sunday is group 'a'
         where uw_housing_group = CHR(EXTRACT(DOW from current_date)::int + 97)
+        -- Filter to participants who are not in an added surveillance group becuase they already get invited weekly
+        and added_surveillance_groups is null
         -- Filter to encounters for participants whose last invite was before today
         and (latest_invite_date is null or latest_invite_date < current_date)
 	),
 
+  -- Select added_surveillance_groups if they are in the group for today
+    added_surveillance_groups as (
+        select
+            redcap_url,
+            redcap_project_id,
+            redcap_record_id,
+            redcap_event_name,
+            redcap_repeat_instance,
+            encountered,
+            individual,
+            tier,
+            latest_invite_date,
+            latest_collection_date,
+            2 as priority,
+            'surveillance' as priority_reason,
```

Comment:
I just want to confirm my understanding that this new surveillance group has the same priority as uw_housing_resident. 

Additionally would it be confusing for people if some surveillance have priority of 2 while other surveillance have priority of 5.

Reply:
That's right. This new group has a priority of 2 to increase the odds that everybody who should get invited on a day gets invited. We have enough quota every day to get through every group except surveillance, priority 5.
While this group has priority 2, it's labeled as "surveillance". From the REDCap point of view on the testing determination form, there won't be a distinction between PTs invited for regular surveillance and for this new added surveillance. See Slack thread https://seattle-flu-study.slack.com/archives/C018V0WGGJ0/p1615327884004000